modules/packages/PythonHelper/ArrayTypes.c

```diff
@@ -52,13 +105,45 @@ chpl_ARRAY_TYPES(chpl_MAKE_REPR)
 
 // TODO: how can we remove the error checking here with --no-checks or -scheckExceptions=false?
 #define chpl_MAKE_SETITEM(DATATYPE, CHAPELDATATYPE, NAMESUFFIX, CHECKTYPE, ASTYPE, _0) \
-  static int Array##NAMESUFFIX##Object_setitem(Array##NAMESUFFIX##Object* self, Py_ssize_t index, PyObject* value) { \
+  static int Array##NAMESUFFIX##Object_sq_setitem(Array##NAMESUFFIX##Object* self, Py_ssize_t index, PyObject* value) { \
```

Comment:
What do `sq` and `mp` stand for in `Object_sq_setitem`?  Would it make sense to leave a comment about it?

Reply:
`sq` is for sequence and `mp` is for mapping.  These shortenings come from the `Py_sq_*` and `Py_mp_*` type slots.